src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -231,7 +234,11 @@ public void verifyValMorgan(String continent, String country, Boolean isMobile)
     String MEGA_BLB;
 
     if (!isMobile) {
-      MEGA_LB = VAL_MORGAN_TLB_MEGA_AD_UNIT;
+      if (country.equals("NZ")) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the "NZ" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1132)